diesel_cli/src/print_schema.rs

```diff
@@ -275,7 +279,14 @@ impl Display for CustomTypeList {
                         writeln!(out, "///")?;
                         writeln!(out, "/// (Automatically generated by Diesel.)")?;
                     }
-                    writeln!(out, "#[derive(diesel::sql_types::SqlType)]")?;
+                    if self.derives.is_empty() {
+                        writeln!(
+                            out,
+                            "#[derive(diesel::sql_types::SqlType, diesel::query_builder::QueryId)]"
+                        )?;
```

Comment:
Didn't we say it was breaking if we were doing `QueryId` by default and that was why we wanted to have a config that people would instead update manually?